contracts/token/ERC7984/extensions/ERC7984Freezable.sol

```diff
@@ -51,15 +58,51 @@ abstract contract ERC7984Freezable is ERC7984 {
     /// @dev Unimplemented function that must revert if `msg.sender` is not authorized as a freezer.
     function _checkFreezer() internal virtual;
 
```

Comment:
Given that we no longer have public setters, I think we should remove `_checkFreezer`. This should have been done in the PR that removed the setters.